string obj_class
float64 dist
float64 err_x_m
float64 err_y_m
int64 err_x_pix
int64 err_y_pix
uint64 res_w
uint64 res_h
